activerecord/lib/active_record/store.rb

```diff
@@ -39,14 +43,20 @@ module ActiveRecord
   #   u.settings[:country]  # => 'Denmark'
   #   u.settings['country'] # => 'Denmark'
   #
+  #   # Dirty tracking
+  #   u.color = 'green'
+  #   u.color_changed? # => true
+  #   u.color_was # => 'black'
+  #   u.color_change # => ['black', 'red']
+  #
   #   # Add additional accessors to an existing store through store_accessor
   #   class SuperUser < User
   #     store_accessor :settings, :privileges, :servants
```

Comment:
These'll also get change tracking too, right?

Reply:
Could you please explain, what did you mean here?

Reply:
It's basically what you demonstrate in your PR description. I'm just asking if `store_accessor :color` also adds `color_changed?` etc. or if we missed it and only `store :settings, accessors: %i( color )` would get it.